app/uploaders/photo_uploader.rb

```diff
@@ -0,0 +1,64 @@
+class PhotoUploader < CarrierWave::Uploader::Base
+
+  include Cloudinary::CarrierWave
+
+  process eager: true # Force version generation at upload time.
+
+  process convert: 'jpg'
+
+  version :thumnail do
+    resize_to_fit 256, 256
+  end
+
```

Comment:
`version :avatar do
    cloudinary_transformation transformation: [
      { width: 400, height: 400, gravity: :face, radius: :max, crop: :thumb },
      { width: 200, height: 200, crop: :thumb }
    ]
  end`

Reply:
added in the code